server/libs/infra/src/typesense-schemas/asset.schema.ts

```diff
@@ -13,6 +13,7 @@ export const assetSchema: CollectionCreateSchema = {
     { name: 'fileCreatedAt', type: 'string', facet: false, sort: true },
     { name: 'fileModifiedAt', type: 'string', facet: false, sort: true },
     { name: 'isFavorite', type: 'bool', facet: true },
+    { name: 'isArchived', type: 'bool', facet: true },
```

Comment:
We can leave, but we might not need to ever filter by this via typesense, as a database query is sufficient.

Reply:
Removed `isArchived` from typesense schema